tests/test_bundle.py

```diff
@@ -7,11 +7,13 @@
 
 import lightkube.codecs
 import pytest
+import pytest_asyncio
```

Comment:
We should probably add this as a requirement in `knative-operators/requirements-integration.in`, as we are explicitly calling it in the test.py file.

Reply:
added